src/GitLink/Providers/BitBucketProvider.cs

```diff
@@ -13,21 +12,18 @@ namespace GitLink.Providers
 
     public class BitBucketProvider : ProviderBase
     {
-        private readonly Regex _gitHubRegex = new Regex(@"(?<url>(?<companyurl>(?:https://)?bitbucket\.org/(?<company>[^/]+))/(?<project>[^/]+))");
+        private static readonly Regex HostingUrlPattern = new Regex(@"(?<url>(?<companyurl>(?:https://)?bitbucket\.org/(?<company>[^/]+))/(?<project>[^/]+))");
```

Comment:
Great change, I recently noticed this and it immediately bothered me. This is great for the next provider where we copy/paste ;-)